backend/fms_core/migrations/0074_profile_freezemanuser.py

```diff
@@ -0,0 +1,100 @@
+        reversion.set_comment(f"Create FreezemanUser for all existing users.")
+        reversion.set_user(admin_user)
+
+        for user in User.objects.all():
+            fm_user = FreezemanUser.objects.create(
+                user_id=user.id,
+                profile=default_profile,
+                created_by_id=admin_user.id,
+                updated_by_id=admin_user.id,
+            )
+            reversion.add_to_revision(fm_user)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('fms_core', '0073_v5_3_0'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Profile',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True, help_text='Date the instance was created.')),
+                ('updated_at', models.DateTimeField(auto_now=True, help_text='Date the instance was modified.')),
+                ('deleted', models.BooleanField(default=False, help_text='Whether this instance has been deleted.')),
+                ('name', models.CharField(help_text="Name of the profile (e.g. 'TechDev', 'Production Lab', username, etc.)", max_length=50, unique=True)),
+                ('preferences', models.JSONField(default=dict, help_text='Preferences stored as a JSON object')),
+                ('created_by', models.ForeignKey(blank=True, on_delete=django.db.models.deletion.PROTECT, related_name='%(app_label)s_%(class)s_creation', to=settings.AUTH_USER_MODEL)),
+                ('parent', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.PROTECT, to='fms_core.profile')),
+                ('updated_by', models.ForeignKey(blank=True, on_delete=django.db.models.deletion.PROTECT, related_name='%(app_label)s_%(class)s_modification', to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='FreezemanUser',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True, help_text='Date the instance was created.')),
+                ('updated_at', models.DateTimeField(auto_now=True, help_text='Date the instance was modified.')),
+                ('deleted', models.BooleanField(default=False, help_text='Whether this instance has been deleted.')),
+                ('created_by', models.ForeignKey(blank=True, on_delete=django.db.models.deletion.PROTECT, related_name='%(app_label)s_%(class)s_creation', to=settings.AUTH_USER_MODEL)),
+                ('profile', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='users', to='fms_core.profile')),
```

Comment:
maybe user freezeman_users as related_name to make sure there is no confusion when we write in code.